apps/ledger-live-mobile/android/app/src/main/jni/MainApplicationModuleProvider.cpp

```diff
@@ -0,0 +1,20 @@
+#include "MainApplicationModuleProvider.h"
```

Comment:
This file isn't used yet as we are not activating the new RN architecture (Fabric + TurboModules)